app/build.gradle

```diff
@@ -31,8 +31,8 @@ android {
     compileSdkVersion 29
     buildToolsVersion "29.0.2"
     defaultConfig {
-        applicationId "io.bluetrace.opentrace"
-        resValue "string", "build_config_package", "io.bluetrace.opentrace"
+        applicationId "com.kccreate.intelligentlockdown"
```

Comment:
should we uncomment line 89 in this file and change it to `dev` to match the development applicationId?

Reply:
we would need to update Firebase first

Reply:
okay , let's go with this for now and add/modify when necessary